src/index.html

```diff
@@ -7,7 +7,50 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="cover">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
```

Comment:
```suggestion
      </div>
      
      <div class="stars stars--1">
```
[checklist p.1 ](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

Reply:
Thank you! Made all the changes, ready for checking